### ISSUE TITLE
Problem meshing periodic 3D structure (Meshing of poly 8 (8) failed)

### ISSUE THREAD

**Issue by msedlak77** [rquey/neper#118]
Starting with:
```
$ neper -T -n 1000 -morpho graingrowth -periodicity x,y
```
Works fine. Then:
```
$ neper -M n1000-id1.tess -format inp

========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 4.1.2
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt
Info   : Running on 8 threads.
Info   : <https://neper.info>
Info   : Copyright (C) 2003-2020, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/home/micke/.neperrc').
Info   : ---------------------------------------------------------------
Info   : MODULE  -M loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] n1000-id1.tess -format inp
Info   : ---------------------------------------------------------------
Info   : Reading input data...
Info   :   - Reading arguments...
Info   : Loading input data...
Info   :   - Loading tessellation...
Info   :     [i] Parsing file `n1000-id1.tess'...
Info   :     [i] Parsed file `n1000-id1.tess'.
Info   : Meshing...
Info   :   - Preparing... (cl = 0.05) 100%
Info   :   - 0D meshing... 100%
Info   :   - 1D meshing... 100%
Info   :   - 2D meshing... 100% (0|0.62/97%| 3%| 0%)     
Info   :   - Fixing 2D-mesh pinches...
Info   :   - 3D meshing... Error  :     > Meshing of poly 8 (8) failed
Aborted (core dumped)
```

I have tried different element size, without success. 
Thanks in advance!

**Comment by rquey** [comment]
3D meshing seems to be broken. How did you install Gmsh?

**Comment by msedlak77** [comment]
Thanks for the fast reply.
When I posted I used version 4.1.2.
The 3D meshing worked for "neper -T -n 1000 -morpho voronoi " and "neper -T -n 1000 -morpho graingrowth"
but it did not work for any periodicity (x,xy,all).

So I have switched to version 3.5.2 here 
neper -T -n 1000 -morpho voronoi works with periodicity (x,y). :)
Unfortinutly I can't get 
neper -T -n 1000 -morpho graingrowth to work with periodicity (x,y). Issue is listed below:

========================    N   e   p   e   r    =======================
Info   : A software package for polycrystal generation and meshing.
Info   : Version 3.5.2
Info   : Built with: gsl|muparser|opengjk|openmp|nlopt
Info   : Running on 8 threads.
Info   : <http://neper.info>
Info   : Copyright (C) 2003-2020, and GNU GPL'd, by Romain Quey.
Info   : No initialization file found (`/home/micke/.neperrc').
Info   : ---------------------------------------------------------------
Info   : MODULE  -M loaded with arguments:
Info   : [ini file] (none)
Info   : [com line] n1000-id1.tess -format inp
Info   : ---------------------------------------------------------------
Info   : Reading input data...
Info   :   - Reading arguments...
Info   : Loading input data...
Info   :   - Loading tessellation...
Info   :     [i] Parsing file `n1000-id1.tess'...
Info   :     [i] Parsed file `n1000-id1.tess'.
Info   : Meshing...
Info   :   - Preparing... (cl = 0.05) 100%
Info   :   - 0D meshing... 100%
Info   :   - 1D meshing... 100%
Info   :   - 2D meshing... 100% (0|0.62/97%| 3%| 0%)     
Info   :   - Fixing 2D-mesh pinches...
Info   :   - 3D meshing... 100% (0|0.77/92%| 8%| 0%)Error  :     > Meshing of poly 953 (953) failed
Aborted (core dumped)

**Comment by rquey** [comment]
This has been fixed in 2dccc1d33828d4b75da649c3f2bdc410274fa8bd.
